rules-reviewed/quarkus/springboot/springboot-parent-pom-to-quarkus.windup.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="springboot-parent-pom-to-quarkus"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This ruleset gives hints to replace the Spring Parent POM with Quarkus BOM
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,3.0.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-java,3.0.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-xml,3.0.0.Final" />
+        </dependencies>
+        <sourceTechnology id="springboot" />
+        <targetTechnology id="quarkus" />
+    </metadata>
+    <rules>
+        <rule id="springboot-parent-pom-to-quarkus-00000">
+            <when>
+                <xmlfile matches="/m:project/m:parent[m:groupId/text() = 'org.springframework.boot' and m:artifactId/text() = 'spring-boot-starter-parent']" in="pom.xml">
```

Comment:
If there are other springboot parent artifacts can we add a conditon for those here also?

Reply:
Good point!
In terms of Spring Boot it seems like there's another one from this search https://search.maven.org/search?q=p:pom%20g:org.springframework.boot%20a:spring*parent
So there's also a `spring-boot-parent` artifact https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-parent
I think it's worth adding this one as well for the sake of covering all the parents for Spring Boot.